scripts/README.md

```diff
@@ -0,0 +1,11 @@
+# Scheduler scripts
+
+1. download_programs.py 
+
+Downloads program needed for the Scheduler to work. 
+DEFAULT_PROGRAMS variable specifies the list of programs the script is going to download
+Programs (the .json.gz file) needs to be on the `data/` for the Provider to work properly.
+
+2. odb.extractor_atoms.py
+
+Script that generates atom for OCS programs. This is Bryan experimental work and does not use current Minimodel structures. 
```

Comment:
Great! I'd love so see us all add a bit more of this. (I know I'm terrible at it.)